### ISSUE TITLE
Allow integers to be typed as enums

### ISSUE THREAD

**Issue by rivatove** [arthurfiorette/prisma-json-types-generator#213]
Hey! Thanks for the library, it works great.

I think the following would be a cool feature.

```prisma
/// [PaymentStatus]
status        Int       @default(0) @db.TinyInt
```

```ts
enum PaymentStatus {
  Unpaid = 0,
  Paid = 1,
}
```

**Comment by arthurfiorette** [comment]
Hey @rivatove this is a awesome feature request! Any reason on why prisma enums cannot be used?

**Comment by rivatove** [comment]
> Hey @rivatove this is a awesome feature request! Any reason on why prisma enums cannot be used?

Hey, this is funny. I actually forgot these exist because the last time I checked my DB engine didn't support them.

I successfully migrated to DB native enums and they fulfill this use case. 

Enums in code have certain advantages such as portability (only integer support is needed) and flexibility (make changes without migrating the database) but I'm not sure if that makes them worth pursuing for the library.

Thanks again!
